swap/protocol.go

```diff
@@ -105,7 +105,7 @@ func (s *Swap) run(p *p2p.Peer, rw p2p.MsgReadWriter) error {
 	protoPeer := protocols.NewPeer(p, rw, Spec)
 
 	handshake, err := protoPeer.Handshake(context.Background(), &HandshakeMsg{
-		ContractAddress: s.owner.Contract,
+		ContractAddress: s.issuer.Contract,
```

Comment:
This is exactly what I meant this morning in the call. 

This is the handshake. At this point, two nodes are just exchanging contract addresses. We don't even know what balances there are (they might be 0 on both sides at this point!). Thus, I see it as incorrect to change this to `issuer`. No cheques may have been issued at this point, thus no issuer exists. Furthermore, at this point both nodes could be the issuer of cheques! I think it does not make sense to introduce this change here.

Reply:
i agree. 

if we don't want to use `owner` for reasons of consistency with the smart contracts, we can find another word.

in any case, the golang context is different from the SC context, so it wouldn't bother me if we were to use additional terms on either side that the other side doesn't.

i'd just insist on using the same terms when referring to the same entities, specially in variables or fields.